backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5771470_sys_gh25448_AD_Table_Changelog_Config_Table.sql

```diff
@@ -0,0 +1,27 @@
+INSERT INTO fix.AD_Table_Changelog_Config_26092025 (I_ChangeLog_Config_id,
+                                    AD_Client_ID,
+                                    AD_Org_ID,
+                                    AD_Table_ID,
+                                    Created,
+                                    CreatedBy,
+                                    IsActive,
+                                    KeepChangeLogsDays,
+                                    Updated,
+                                    UpdatedBy)
+
+SELECT NEXTVAL('AD_ChangeLog_Config_seq'),
```

Comment:
this is not ok to use the seq of another table. And you don't need this. Please, create the fix table as I wrote in my example. it doesn't need any I_ChangeLog_Config_id.
If in doubt, please, take a look how the other fix tables look like

Reply:
For me it's fine to add all except the ID, the ID should be moved to the insert into AD_Changelog_Config. 
(For next time: you can also do CREATE TABLE fix.mytable AS SELECT 1000000 AS ad_client_id, ...)